sources/api/schnauzer/src/helpers.rs

```diff
@@ -2701,6 +2795,8 @@ mod test_any_enabled {
 
 #[cfg(test)]
 mod test_oci_spec {
+    use super::Containerd;
+    use super::Docker;
```

Comment:
It's common to do this. But either way is fine. Though style-wise we would usually see multiple symbols from the same mod like this `use super::{Containerd, Docker};`

```suggestion
    use super::*
```